src/components/IconToolbar.jsx

```diff
@@ -27,14 +27,13 @@ export class IconToolbar extends React.Component {
         const { isActive } = this.state;
         return (
             <div className={setClass(className, { hidden: isHidden })} data-name={title}>
-                <RoundIconButton
+                <SquareIconButton
```

Comment:
What was the point of making this change? It distracts from important changes and doesn't introduce new functionality - dropdown is what's important in this PR